packages/api-clients/open-api/catalogApi.yml

```diff
@@ -2374,6 +2374,9 @@ components:
         instanceLabel:
           type: string
           format: uuid
+        templateVersionId:
+          type: string
+          format: uuid
```

Comment:
The template version ID should not be returned in `EServiceTemplateRef` because it's not related to the e-service but to an e-service descriptor. 
Also, no need to add it to any API response: it's already present inside each descriptor in `templateVersionRef` and you should retrieve it from the descriptor when you need it in the BFF.

```suggestion
```


A side note on this topic, for future PRs: when you add a field in the spec, the only thing that happens is that it is added to the generated models - but then you also need to add some implementation that actually serializes the corresponding value. Otherwise, at runtime, it will probably be `undefined` because you did not add it to serialization (unless there is a spread that copies all the fields).